framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java

```diff
@@ -147,27 +146,27 @@ public void run() {
       } catch (Exception e) {
         LOG.error("Failed while running cleanup query. Not returning connection to pool.", e);
         try {
-			connection.close();
-		} catch (SQLException e1) {
-			LOG.warn(e.getMessage());
-			e1.printStackTrace();
-		}
+			    connection.close();
+		    } catch (SQLException e1) {
+			    LOG.warn(e.getMessage());
+			    e1.printStackTrace();
+		    }
       }
       if (testStatus == TestStatus.PASS && !TestDriver.cmdParam.outputQueryResult) {
-    	Utils.deleteFile(outputFilename);
+    	  Utils.deleteFile(outputFilename);
       }
       duration = stopwatch;
 
-      if(++noOfCasesCompleted%100==0 && noOfCasesCompleted <= totalCases){
-	LOG.info("----------------------------------------------------------------------------------------------------------------");
-
-        LOG.info("Execution completed for "+(noOfCasesCompleted)+" out of "+(totalCases)+" tests");
-	LOG.info("----------------------------------------------------------------------------------------------------------------\n");
-      }
       if (TestDriver.driverType == TestDriver.DriverType.APACHE) {
-        LOG.info(testStatus + " (" + stopwatch + ") " + modeler.queryFilename + " (connection: " + connection.hashCode() + ")" + " (queryID: " + queryID + ")");
+        LOG.info("[" + testStatus + "] (" + stopwatch + ") " + modeler.queryFilename + " (ConnectionID: " + connection.hashCode()+ " | QueryID: " + queryID + ")");
       } else {
-        LOG.info(testStatus + " (" + stopwatch + ") " + modeler.queryFilename + " (connection: " + connection.hashCode() + ")");
+        LOG.info("[" + testStatus + "] (" + stopwatch + ") " + modeler.queryFilename + " (ConnectionID: " + connection.hashCode() + ")");
+      }
+
+      if((++countTestsCompleted %100==0 && countTestsCompleted <= totalCases) || (countTestsCompleted == totalCases)){
```

Comment:
Fixed the positioning of the test progress message to after the dependent test status is printed to console - this way it's accurate.

Also prints the message after all tests are complete.